appGeo/src/tests/game.spec.ts

```diff
@@ -0,0 +1,298 @@
+// import {Chat} from '../app/chat/chat_class_declaration';
```

Comment:
I think all general changes to these tests will be overlapping with #55, which I think has a bit more deeper of an update on these. So we should discuss the connection of the two of these since we obviously shouldn't merge both game testing files.

Reply:
I suppose that #55 will be merged first and I will have to deal with the inevitable merge conflicts that appear. I think that what I will do is just move the tests of Game & Killing Handlers directly over to whatever comes through as a result of #55 being pushed to main. This should avoid headaches and wouldn't break the passing tests.